metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -111,26 +119,22 @@ def extract_dbt_entities(
 
     dbt_entities = []
     for key, node in all_manifest_entities.items():
-        dbtNode = DBTNode()
-
         # check if node pattern allowed based on config file
         if not node_type_pattern.allowed(node["resource_type"]):
             continue
-        dbtNode.dbt_name = key
-        dbtNode.database = node["database"]
-        dbtNode.schema = node["schema"]
-        dbtNode.dbt_file_path = node["original_file_path"]
-        dbtNode.node_type = node["resource_type"]
-        dbtNode.max_loaded_at = sources_by_id.get(key, {}).get("max_loaded_at")
+
+        name = node["name"]
+
         if "identifier" in node and not load_catalog:
-            dbtNode.name = node["identifier"]
-        else:
-            dbtNode.name = node["name"]
+            name = node["identifier"]
+
+        materialization = None
+        upstream_urns = []
 
         if "materialized" in node["config"].keys():
             # It's a model
-            dbtNode.materialization = node["config"]["materialized"]
-            dbtNode.upstream_urns = get_upstreams(
+            materialization = node["config"]["materialized"]
```

Comment:
There will always be "config" key here, right?

Reply:
Not always – switching to `.get()`